test/controllers/album.js

```diff
@@ -155,4 +155,67 @@ describe('/POST albums', () => {
 
   });
 
+});
+
+/*
+* Testing the /user/albums/photos (GET) route
+*/
+describe('/GET users/albums/photos', () => {
+
+  beforeEach(() => {
+    User.create(newUser).then(user => {
+      Album.create({id: 1, userId: user.id, title: 'test'});
+    });
+  });
+
+  it('should successfully retrieve all the photos of a purchased album', (done) => {
+
+    chai.request(server)
+      .post('/users/sessions')
+      .send(correctLogin)
+      .then(auth => {
+        chai.request(server)
+          .get('/users/albums/photos?id=1')
```

Comment:
In this route it seems you are asking for the photo with id 1, instead of the photos of the album with id 1. Thas why it would fit better if the route is `/users/albums/:albumId/photos`